src/components/modals/PlaceOrderModal.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
```

Comment:
🟢 Looks great! But again , maybe we can move all  "magic numbers" to Tailwind config.